daal4py/sklearn/test/test_run_to_run_stability_tests.py

```diff
@@ -318,7 +318,6 @@ def _run_test(model, methods, dataset):
     'RandomForestRegressor',  # will be fixed for next release
     'LinearRegression',  # Absolute diff is 1e-12, will be fixed for next release
     'Ridge',  # Absolute diff is 1e-12, will be fixed for next release
-    'SVC',  # Absolute diff is 1e-9, will be fixed for next release for predict

```

Comment:
I think you shouldn't change `TO_SKIP`. Update [`stable_algos` list](https://github.com/IntelPython/daal4py/blob/f0b886c1f8de5648c3b7c69bec1629c072148779/daal4py/sklearn/test/test_run_to_run_stability_tests.py#L326-L328) instead

Reply:
No, because in oneDAL 2021.1 the algorithm was stable